inlineplz/linters/prospector.py

```diff
@@ -5,10 +5,42 @@
 import traceback
 
 import dirtyjson as json
+import sys
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import sys" should be placed before "import dirtyjson as json" (wrong-import-order)`